ts/addTodo.ts

```diff
@@ -1,10 +1,10 @@
 import { todos } from "./todoapp.js";
 
-function addTodo(todo) {
+function addTodo(todo: { name: string; id: number; completed: boolean }) {
```

Comment:
You repeat this type several times. This type is about Todo.

So create `types.ts` module which will be shared module, meaning that other modules can consume types from it.
Create `Todo` type and reshare between others